server/middleware.go

```diff
@@ -456,6 +456,7 @@ func requireGroup(groupRepo datastore.GroupRepository, cache cache.Cache) func(n
 					}
 				}
 			} else {
+				// TODO(all): maybe we should only use default-group if require_auth is false?
```

Comment:
I think we should not even have the `require_auth` config option

Reply:
@Subomi see this

Reply:
Yes, I agree. Let's eliminate it. @danvixent

Reply:
Obviously in another PR. @danvixent

Reply:
okay